easybuild/easyconfigs/y/YODA/YODA-1.9.7-gompi-2022a.eb

```diff
@@ -0,0 +1,40 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Alexander Puck Neuwirth <alexander@neuwirth-informatik.de>
+
+easyblock = 'ConfigureMake'
+
+name = 'YODA'
+version = '1.9.7'
+
+homepage = 'https://yoda.hepforge.org/'
+description = """
+Yet more Objects for (High Energy Physics) Data Analysis
+"""
+
+toolchain = {'name': 'GCC', 'version': '11.3.0'}
```

Comment:
the filename of the easyconfig should be updated to match the switch to `GCC`

Reply:
if you're using the GitHub integration to update the PR, the syntax would be:
`eb --update-pr 17343 :YODA-1.9.7-gompi-2022a.eb YODA-1.9.7-GCC-11.3.0.eb`

Reply:
Thanks, that is useful. Sorry for this messy PR :/